packages/datetime/src/components/date-picker/date-picker.md

```diff
@@ -1,43 +1,28 @@
----
-tag: new
----
+@# DatePicker
 
-@# DatePicker3
+**DatePicker** renders a UI to choose a single date and (optionally) a time of day.
+It is built on top of the [react-day-picker v8](https://daypicker.dev/v8) library.
+Time selection is enabled by the [TimePicker](#datetime/timepicker) component.
```

Comment:
Do we still care about sharing this information? This feels like is only relevant when comparing datetime and datetime2.

Reply:
Yeah, fair question. I kept the references around since we explicitly mentioned it on the original DatePicker docs: https://github.com/palantir/blueprint/blob/d356c8eea907c1ffc84ef626188c90a3698153fc/packages/datetime/src/components/date-picker/datepicker.md?plain=1#L21-L22

Perhaps we can keep that reference in the updated docs and remove it for the other components.